### ISSUE TITLE
Translate surveys, tasks and fields

### ISSUE THREAD

**Issue by rjmackay** [ushahidi/platform#1923]
- Following on from #1922
- Accept and return translations via existing API endpoints. Structure something like:
```
           {
                "parent_id":null,
                "tag":"Boxes",
                "slug":"boxes",
                "description":"Is this a box? Awesome",
                "type":"category",
                "priority":1,
                "color":"00ff00",
                "role": [],
                "translations": {
                      "ar": { "tag": "Arabic for boxes"}
                }
            }
```
- Update survey settings to add extra fields to configure translations
- Update Forms, Stages and Attributes endpoints to use translations depending on current language

**Comment by Celestine22** [comment]
Option below selected by client.
@rjmackay @willdoran @jshorland 

*Option 2*
--

- [ ] Build in translations for categories, site name, site description as in option 1.Build in translations for surveys, as well tasks and fields within the survey.Take a similar approach to translating tags for survey title, tasks and fields.
- [ ] Create only a single survey for Arabic and English (so just 1 survey, or 3 surveys if we are using separate surveys per domain).When configuring a survey you would see 'English Title' 'Arabic title' inputs, this is also similar for when admin is configuring the tasks and fields.
- [ ] When a survey is shown on the frontend we switch English/Arabic labels for inputs.When filtering by survey, we show all results regardless of language.
- [ ] The user can filter the incidents based on the environmental domain, the map will show the incidents of the selected domain regardless the selected language.
- [ ] Additional 2.5 weeks of work
- [ ] 
- [ ]

**Comment by rjmackay** [comment]
This issue was moved to ushahidi/platform-fix-our-environment#25

**Comment by rjmackay** [comment]
This issue was moved to ushahidi/platform-fix-our-environment#26
